src/main/java/org/symphonyoss/symphony/messageml/elements/TextField.java

```diff
@@ -8,7 +8,12 @@
 import org.w3c.dom.NamedNodeMap;
 import org.w3c.dom.NodeList;
 
-import java.util.*;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Set;
 
 
 /**
```

Comment:
Some more explanation about the behaviour of the "masked" attribute and performed validations could be useful.